app/src/main/java/io/github/hidroh/materialistic/WebFragment.java

```diff
@@ -445,6 +479,9 @@ void setFullscreen(boolean isFullscreen) {
             params.height = ViewGroup.LayoutParams.MATCH_PARENT;
         } else {
             reset();
+            if (mWebView.getUrl().equals(PDF_LOADER_URL)) {
+                mWebView.setInitialScale(1);
+            }
```

Comment:
Let's remove the if condition here and force reset scale when we toggle between fullscreen & window mode?

Reply:
Sure, though apparently `setInitialScale` doesn't work, I should use `zoomOut` here.